src/engine.ts

```diff
@@ -470,49 +473,30 @@ export const wanderingNCs = new Set<string>([
   "Teacher's Pet",
   // Lil' Doctor™ bag noncombat
   "A Pound of Cure",
-  // Turtle taming noncombats
-  "Nantucket Snapper",
-  "Blue Monday",
-  "Capital!",
-  "Training Day",
-  "Boxed In",
-  "Duel Nature",
-  "Slow Food",
-  "A Rolling Turtle Gathers No Moss",
-  "Slow Road to Hell",
-  "C'mere, Little Fella",
-  "The Real Victims",
-  "Like That Time in Tortuga",
-  "Cleansing your Palette",
-  "Harem Scarum",
-  "Turtle in peril",
-  "No Man, No Hole",
-  "Slow and Steady Wins the Brawl",
-  "Stormy Weather",
-  "Turtles of the Universe",
-  "O Turtle Were Art Thou",
-  "Allow 6-8 Weeks For Delivery",
-  "Kick the Can",
-  "Turtles All The Way Around",
-  "More eXtreme Than Usual",
-  "Jewel in the Rough",
-  "The worst kind of drowning",
-  "Even Tamer Than Usual",
-  "Never Break the Chain",
-  "Close, but Yes Cigar",
-  "Armchair Quarterback",
-  "This Turtle Rocks!",
-  "Really Sticking Her Neck Out",
-  "It Came from Beneath the Sewer? Great!",
-  "Don't Be Alarmed, Now",
-  "Puttin' it on Wax",
-  "More Like... Hurtle",
-  "Musk! Musk! Musk!",
-  "Silent Strolling",
+]);
+
+export const environmentSpecificNCs = new Map<string, EnvironmentType>([
+  ["Even Tamer Than Usual", "indoor"],
+  ["Never Break the Chain", "indoor"],
+  ["Close, but Yes Cigar", "indoor"],
+  ["Armchair Quarterback", "indoor"],
+  ["This Turtle Rocks!", "outdoor"],
+  ["Really Sticking Her Neck Out", "outdoor"],
+  ["It Came from Beneath the Sewer? Great!", "outdoor"],
+  ["Don't Be Alarmed, Now", "outdoor"],
+  ["Puttin' it on Wax", "underground"],
+  ["More Like... Hurtle", "underground"],
+  ["Musk! Musk! Musk!", "underground"],
+  ["Silent Strolling", "underwater"],
 ]);
 
 export const zoneSpecificNCs = new Map<string, Location[]>([
   ["The Horror...", $locations`Frat House`], // Duplicate choice name
+  ...fileToBuffer("data/encounters.txt")
```

Comment:
"The Horror" above can probably also be deleted, assuming it is part of this encounters file.